### PR TITLE
News and announcements

### PR DESCRIPTION
**Pull request by SaranjeetKaur**
This chapter provides resources that can be useful to keep up with the developments in R.

### CODE REVIEW

**Comment by SaranjeetKaur** [comment]
The chapter number will be provided later.

File: news_and_announcements.Rmd [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# News and Announcements
+
+Here are some resources that can be useful to keep up with the developments in R:
+
+## Blogs
+
+A list of public [blogs](https://developer.r-project.org/Blog/public/) mainly written by the R Core Development Team. Daily news about R-devel can be found [here](https://developer.r-project.org/blosxom.cgi/R-devel).
```
Comment (willingc) [inline news_and_announcements.Rmd:7]: The R project maintains...

File: news_and_announcements.Rmd [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+A list of public [blogs](https://developer.r-project.org/Blog/public/) mainly written by the R Core Development Team. Daily news about R-devel can be found [here](https://developer.r-project.org/blosxom.cgi/R-devel).
+
+## Conferences
+
+Updates about conferences actively supported or endorsed by The R Foundation can be found [here](https://www.r-project.org/conferences/). These conferences are organised by members from the R community.
+
+## Journal
+
+[The R Journal](https://journal.r-project.org/) is a open access, referred journal featuring short to medium length articles that should be of interest to users or developers of R. It also has a news section where information on, changes in R (new features of the latest release), changes on CRAN (new add-on packages, manuals, binary contributions, mirrors, etc.), upcoming conferences, and conference reports is provided.
```
Comment (willingc) [inline news_and_announcements.Rmd:15]:
```suggestion
[The R Journal](https://journal.r-project.org/) is an open access and refereed journal featuring short to medium length articles that should be of interest to users or developers of R. It also has a news section where information on, changes in R (new features of the latest release), changes on CRAN (new add-on packages, manuals, binary contributions, mirrors, etc.), upcoming conferences, and conference reports is provided.

```

**Review by willingc — COMMENTED** [review]
Thanksl. A few small suggested changes.

File: news_and_announcements.Rmd [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+
+## Mailing lists
+
+  * [R-announce](https://stat.ethz.ch/mailman/listinfo/r-announce): A moderated mailing list used for announcements by the R Core Development Team. Major announcements about the development of R and the availability of new code are made here.
+  * [R-packages](https://stat.ethz.ch/mailman/listinfo/r-packages): A moderated mailing list for announcements about contributed R packages (typically on [CRAN](https://cran.r-project.org/)) and similar R project extensions.
+
+## Twitter
+
+Follow ['@R_dev_news'](https://twitter.com/R_dev_news) on Twitter for the [R development updates](https://developer.r-project.org/RSSfeeds.html) and [blog post](https://developer.r-project.org/Blog/public/) announcements.
```
Comment (llrs) [inline news_and_announcements.Rmd:25]: Thanks for mentioning the account R_dev_news, but it is not official and don't think it should be mentioned here. Instead, as it was pointed out on the mailing list, there should be a mention to the @_R_foundation account, which also provides information about grants, the blog and other relevant content. 
Perhaps there should mention that exists some other accounts which provide the same information from these and other source (I can think about the cranberries about updates on CRAN packages, the account that tracks CRAN policy changes..., R_dev_news). 

The section name could be changed to social media with a section for other social site where news and updates are shared (linkedin, facebook, xing, instagram, tick-tock?).

**Review by llrs — COMMENTED** [review]
I think this is a great chapter very important for people that want to know what is going on. It is great to have a one place to know about all these sources!

File: news_and_announcements.Rmd (hunk 3 above)
Comment (willingc) [reply news_and_announcements.Rmd:25]: Perhaps stick with the official resources and mention at the end that there are other unofficial resources too numerous to list here.

File: news_and_announcements.Rmd [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+The [R project](https://www.r-project.org/) maintains a list of public [blogs](https://developer.r-project.org/Blog/public/) mainly written by the R Core Development Team. Daily news about R-devel can be found [here](https://developer.r-project.org/blosxom.cgi/R-devel).
+
+## Conferences
+
+Updates about conferences actively supported or endorsed by The R Foundation can be found [here](https://www.r-project.org/conferences/). These conferences are organised by members from the R community.
+
+## Journal
+
+[The R Journal](https://journal.r-project.org/) is an open access and refereed journal featuring short to medium length articles that should be of interest to users or developers of R. It also has a news section where information on, changes in R (new features of the latest release), changes on CRAN (new add-on packages, manuals, binary contributions, mirrors, etc.), upcoming conferences, and conference reports is provided.
```
Comment (MichaelChirico) [inline news_and_announcements.Rmd:15]: > information on,

sentence fragment

**Pull request merged**